src/guide-dist.md

```diff
@@ -78,6 +78,27 @@ Lets go over the distributions by type:
     [`Uniform`] (for the latter, `low` and `high` parameters are *also* SIMD
     types, effectively sampling from multiple ranges simultaneously). SIMD
     support is gated behind a [feature flag](../features.html#simd-support).
+-   For enums, you have to implement uniform sampling yourself. For example, you
+    could use the following approach:
```

Comment:
Perhaps we should mention at least the `EnumIter` option, or reference #644? The below would be a nuisance to maintain with large enums.